src/core/Button/style.ts

```diff
@@ -8,16 +8,33 @@ import {
   Props,
 } from "../styles";
 
-const sdsPropNames = ["isAllCaps", "isRounded", "sdsStyle", "sdsType"];
+export interface ExtraProps {
+  isAllCaps?: boolean;
+  isRounded?: boolean;
+  /**
+   * TODO(185930): Remove custom `color` prop when we upgrade to MUIv5.
```

Comment:
@mdjmoore I forgot we haven't merged the button PR 😂 

Also created a ticket already!

https://app.shortcut.com/sci-design-system/story/185930/material-ui-mui-v5-todos

Thank you!